int32 code
int32 n_poste
